test/model/test_controller.py

```diff
@@ -306,6 +318,52 @@ async def test_begin_inclusion_errors(controller):
             InclusionStrategy.SECURITY_S2, force_security=True
         )
 
+    ack_commands = mock_command(
+        {"command": "controller.provision_smart_start_node"},
+        {"success": True},
+    )
+
+    provisioning_entry = controller_pkg.QRProvisioningInformation(
+        QRCodeVersion.SMART_START,
+        [SecurityClass.S2_UNAUTHENTICATED],
+        "test",
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
```

Comment:
Make these values unique (ie incremental) so you can verify the inputs are mapped to correct msg key. Using keyword args here would also help verify that things work. Having so many positional args is easy to mess up.